aten/src/ATen/cuda/CUDAGraph.h

```diff
@@ -0,0 +1,30 @@
+#include <ATen/Tensor.h>
+#include <c10/core/Device.h>
+#include <c10/cuda/CUDAStream.h>
+
+namespace at {
+namespace cuda {
+
+struct TORCH_CUDA_API CUDAGraph {
```

Comment:
Probably should have a `// See Note [CUDA Graph...` reference somewhere here; probably framed as "why does PyTorch need its own CUDAGraph rep"

Reply:
Added [Note [CUDA Graph Wrapper Class]](https://github.com/pytorch/pytorch/pull/48875/files#diff-d7302d133bb5e0890fc94de9aeea4d9d442555a3b40772c9db10edb5cf36a35cR10-R23)